popup.html

```diff
@@ -28,6 +28,9 @@
         background-image: linear-gradient(0deg, #fff, #fff, #f1f1f1);
         padding: 8px;
       }
+      #loginPrompt {
+        color: #c2a90f;
```

Comment:
Looking at it now, dark yellow doesn't look as good as red. Sorry about that! I would make it `#cc0000` instead.